mlrun/runtimes/function.py

```diff
@@ -393,11 +425,17 @@ def with_http(
                 raise ValueError(
                     "gateway timeout must be greater than the worker timeout"
                 )
-            annotations["nginx.org/proxy-connect-timeout"] = f"{gateway_timeout}s"
-            annotations["nginx.org/proxy-read-timeout"] = f"{gateway_timeout}s"
-            annotations["nginx.org/proxy-send-timeout"] = f"{gateway_timeout}s"
+            annotations[
+                "nginx.ingress.kubernetes.io/proxy-connect-timeout"
+            ] = f"{gateway_timeout}"
+            annotations[
+                "nginx.ingress.kubernetes.io/proxy-read-timeout"
+            ] = f"{gateway_timeout}"
+            annotations[
+                "nginx.ingress.kubernetes.io/proxy-send-timeout"
+            ] = f"{gateway_timeout}"
         self.add_trigger(
-            "http",
+            trigger_name or "http",
```

Comment:
@Hedingber default http trigger name should be http_trigger ( and not http)?

Reply:
If by "should" you mean you want to align it to Nuclio's default behavior - then Nuclio calls it `default-http`